R/later_recurring.R

```diff
@@ -0,0 +1,24 @@
+#' @describeIn later Schedules a recurring task
```

Comment:
Please add documentation about returning `FALSE`.

Reply:
Yes, I was thinking about where to add this. I now see that I can add to `@details` within this file, I'll add a quick blurb.